server/index.js

```diff
@@ -92,6 +117,46 @@ app.get('/api/posts', (req, res, next) => {
     .catch(err => next(err));
 });
 
+app.get('/api/comments', (req, res, next) => {
+  const sql = `
+  select *
+    from "comments"
+  `;
+  db.query(sql)
+    .then(result => {
+      res.json(result.rows);
+    })
+    .catch(err => next(err));
+});
+
+app.get('/api/comments/:commentId', (req, res, next) => {
+  const commentId = Number(req.params.commentId);
+  const params = [commentId];
+  if (!Number.isInteger(commentId) || commentId <= 0) {
+    res.status(400).json({
+      error: '"commentId" must be a positive integer'
+    });
+    return;
+  }
+  const sql = `
+    select *
+      from "comments"
+     where "messageId" = $1
+  `;
+  db.query(sql, params)
+    .then(result => {
+      const post = result.rows[0];
+      if (!post) {
+        res.status(404).json({
+          error: `${commentId} not found`
+        });
+      } else {
+        res.json(post);
+      }
+    })
```

Comment:
I'm pretty sure these routes aren't necessary, just have your `/api/posts` include the comments nested inside each post as mentioned above. Then, use the commend object from the response of your `app.post('/api/comments', (req, res) => {` instead of doing a whole new `fetch()`.